rules.ini

```
@@ -124,6 +124,9 @@ RPGMaker[] = (?:^|/)RPG_RT\.ini$
 RPGMaker[] = \.(?:rgssad|rvproj2|rgss3a|rgss2a)$
 SakanaGL[] = (?:^|/)sakanagl\.dll$
 SakanaGL[] = \.sxstorage$
+SmileGameBuilder[] = (?:^|/)data\.sgbpack$
+SmileGameBuilder[] = /sgb_rpgplayer\.resources\.dll$
+SmileGameBuilder[] = (?:^|/)sgbplog\.txt$
```

no need to try to match this engine's sdk's app
```suggestion
SmileGameBuilder = (?:^|/)data\.sgbpack$
```